### ISSUE TITLE
Review and reorganize training videos

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1207]
This should be done in conjunction with the accessibility changes and changes to the videos to match the new website.  There are videos on YouTube that aren't on onrr.gov currently, so we need to do an audit and make sure we have all the videos and have them organized well.

**Comment by mcharg** [comment]
@daniel-coleman-1 This should be done later after the 508 compliance is done for the videos.

**Comment by mcharg** [comment]
I'm doing this as part of getting to the list of all the videos.

**Comment by mcharg** [comment]
Audit is done. Will revisit organizing once we know what the final set of videos is.
https://doimspp.sharepoint.com/:x:/r/sites/NRRD/Shared%20Documents/General/ONRR.gov/Content/Content%20Audit/ContentTracking.xlsx?d=w74b27b0272bc4cee92b0a1280895cb73&csf=1&web=1&e=5j1UX1
